rca/people/tests/test_models.py

```diff
@@ -275,3 +275,60 @@ def test_create_page_validation_no_collection(self):
             "If you are adding a student user account so a student can access "
             "this page, an image collection must be added.",
         )
+
+
+class TestStudentPageEdit(TestCase, WagtailTestUtils):
+
+    create_view_name = "student_account_create"
+
+    def setUp(self):
+        super().setUp()
+        self.student = UserFactory(username="student")
+        self.user = UserFactory(is_superuser=True)
+        self.student_group = Group.objects.get(name="Students")
+        admin_permission = Permission.objects.get(codename="access_admin")
+        self.student_group.permissions.add(admin_permission)
+        self.student.groups.add(self.student_group)
+        self.student.set_password("test")
+        self.student.save()
+
+        self.home_page = HomePage.objects.first()
+        self.home_page.add_child(
+            instance=StudentIndexPage(
+                title="Students",
+                slug="students",
+                introduction="students",
+            )
+        )
+        self.student_index = StudentIndexPage.objects.first()
+        GroupPagePermission.objects.create(
+            group=self.student_group,
+            page=self.student_index,
+            permission_type="edit",
```

Comment:
Hi @patrickcuagan I needed to revert this work as it was pushed to dev and causing problem with deploying the Wagtail 5.1 upgrade to dev for testing.

My suggestion here is untested but reflects the changes required here: https://docs.wagtail.org/en/stable/releases/5.1.html#grouppagepermission-now-uses-django-s-permission-model

```suggestion
            permission=Permission.objects.get(
                content_type__app_label="wagtailcore", codename="change_page"
            )
```

It may be best to wait until the 5.1 upgrade has been deployed and the rebase your work to see the error in the test.